python/docs/source/getting_started/quickstart_df.ipynb

```diff
@@ -424,76 +424,22 @@
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/html": [
-       "<div>\n",
-       "<style scoped>\n",
-       "    .dataframe tbody tr th:only-of-type {\n",
-       "        vertical-align: middle;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe tbody tr th {\n",
-       "        vertical-align: top;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe thead th {\n",
-       "        text-align: right;\n",
-       "    }\n",
-       "</style>\n",
-       "<table border=\"1\" class=\"dataframe\">\n",
-       "  <thead>\n",
-       "    <tr style=\"text-align: right;\">\n",
-       "      <th></th>\n",
-       "      <th>a</th>\n",
-       "      <th>b</th>\n",
-       "      <th>c</th>\n",
-       "      <th>d</th>\n",
-       "      <th>e</th>\n",
-       "    </tr>\n",
-       "  </thead>\n",
-       "  <tbody>\n",
-       "    <tr>\n",
-       "      <th>0</th>\n",
-       "      <td>1</td>\n",
-       "      <td>2.0</td>\n",
-       "      <td>string1</td>\n",
-       "      <td>2000-01-01</td>\n",
-       "      <td>2000-01-01 12:00:00</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>1</th>\n",
-       "      <td>2</td>\n",
-       "      <td>3.0</td>\n",
-       "      <td>string2</td>\n",
-       "      <td>2000-02-01</td>\n",
-       "      <td>2000-01-02 12:00:00</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>2</th>\n",
-       "      <td>3</td>\n",
-       "      <td>4.0</td>\n",
-       "      <td>string3</td>\n",
-       "      <td>2000-03-01</td>\n",
-       "      <td>2000-01-03 12:00:00</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "   a    b        c           d                   e\n",
-       "0  1  2.0  string1  2000-01-01 2000-01-01 12:00:00\n",
-       "1  2  3.0  string2  2000-02-01 2000-01-02 12:00:00\n",
-       "2  3  4.0  string3  2000-03-01 2000-01-03 12:00:00"
-      ]
-     },
-     "execution_count": 15,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "+----+-----+-----+---------+------------+---------------------+\n",
+      "|    |   a |   b | c       | d          | e                   |\n",
+      "|----+-----+-----+---------+------------+---------------------|\n",
+      "|  0 |   1 |   2 | string1 | 2000-01-01 | 2000-01-01 12:00:00 |\n",
+      "|  1 |   2 |   3 | string2 | 2000-02-01 | 2000-01-02 12:00:00 |\n",
+      "|  2 |   4 |   5 | string3 | 2000-03-01 | 2000-01-03 12:00:00 |\n",
+      "+----+-----+-----+---------+------------+---------------------+\n"
+     ]
     }
    ],
    "source": [
-    "df.toPandas()"
+    "from tabulate import tabulate\n",
+    "print(tabulate(df.toPandas(), headers = 'keys', tablefmt = 'psql'))"
```

Comment:
Hm, the output format looks fine but the whole point of using `spark.sql.repl.eagerEval.enabled` is to show a pretty table format without applying any other operations in the notebook.

Can you maybe just manually fix the output `text/html` to be compatible with both the sphinx dark theme and jupyter notebook?

Reply:
- Dark theme:
   <img width="749" alt="image" src="https://github.com/apache/spark/assets/15246973/95ee1402-8038-4438-808d-e0d02c0fb85a">

- Light theme:
   <img width="738" alt="image" src="https://github.com/apache/spark/assets/15246973/0e3448f3-d455-4136-9a94-ec957463ce21">

Reply:
Oh, I understand what you mean.
For this document, I have modified the style to maintain using `spark.sql.repl.eagerEval.enabled` purpose to show a pretty table format without applying any other operations in the notebook.

Reply:
For `quickstart_ps.`, are we going to do something similar?
Because this example does not use `spark.sql.repl.eagerEval.enabled`.

Reply:
The table format is also correct in `jupyter notebook`
<img width="836" alt="image" src="https://github.com/apache/spark/assets/15246973/602a1b6c-6146-4d69-abab-fc99e2ab5ed8">